### PR TITLE
dawn: fix stdint.h include for arc

### LINKED ISSUE #11686: dawn failing on arc
https://downloads.openwrt.org/snapshots/faillogs/arc_archs/packages/dawn/compile.txt
Maintainer: @PolynomialDivision 

missing stdint.h header looks like.

### PR DESCRIPTION
**Pull request by PolynomialDivision**
Fixes #11686.

### CODE REVIEW

File: net/dawn/Makefile [hunk 1]
```diff
@@ -5,13 +5,13 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=dawn
-PKG_SOURCE_DATE:=2020-03-23
-PKG_RELEASE:=1
+PKG_SOURCE_DATE:=2020-03-28
+PKG_RELEASE:=2
```
Comment (BKPepe) [inline net/dawn/Makefile]: Why?

**Comment by BKPepe** [comment]
Was it compile tested for arc?

File: net/dawn/Makefile (hunk 1 above)
Comment (PolynomialDivision) [reply net/dawn/Makefile]: Because I updated the source?

**Comment by PolynomialDivision** [comment]
> Was it compile tested for arc?
Yes.

File: net/dawn/Makefile (hunk 1 above)
Comment (PolynomialDivision) [reply net/dawn/Makefile]: https://github.com/berlin-open-wireless-lab/DAWN/commit/b967c1e0b68bc05ec5456ab4ecb1a7f0ae75a1b7

Comment (BKPepe) [reply net/dawn/Makefile]: No. Please read [CONTRIBUTING.md](https://github.com/openwrt/packages/blame/master/CONTRIBUTING.md#L36).

**Comment by PolynomialDivision** [comment]
buildfinished

File: net/dawn/Makefile (hunk 1 above)
Comment (PolynomialDivision) [reply net/dawn/Makefile]: But updaing PKG_SOURCE_DATE is correct or?

**Comment by BKPepe** [comment]
I will merge it once buildpr succeed.

File: net/dawn/Makefile (hunk 1 above)
Comment (BKPepe) [reply net/dawn/Makefile]: Yes, updating PKG_SOURCE_DATE is ok.

**Pull request merged**